csfieldguide/curriculum_guides/content/en/apcsp/sections/todo.md

```diff
@@ -4,99 +4,108 @@ Several learning objectives are not currently covered by the CS Field Guide.
+    - **additional wording in end of 3.1 intro to programming-languages** [Phase1][draft]
 - models and simulations (there are some CS unplugged ones)
- - **new chapter on models-simulations**
+    - **new chapter on models-simulations**
 
 ## Algorithms
+
 - sequencing, selection, iteration as building blocks for algorithms
- - **new sub-section with specific terms in section 2.4** [Phase1][draft]
+    - **new sub-section with specific terms in section 2.4** [Phase1][draft]
 - algorithm correctness
- - **new sub-section in 2.1** [Phase1][draft]
+    - **new sub-section in 2.1** [Phase1][draft]
 - decidability
- - **new section in 11.?** [Phase2][draft]
+    - **new section in 11.?** [Phase2][draft]
 - combining algorithms
- - **add terms to 2.1** [Phase1][draft]
+    - **add terms to 2.1** [Phase1][draft]
 - language choice for algorithms
- - **better description at end of section 2.1.1** [Phase1][draft]
+    - **better description at end of section 2.1.1** [Phase1][draft]
 - theoretical vs empirical analysis
- - **additional paragraph in 2.1.2** [Phase2][draft]
+    - **additional paragraph in 2.1.2** [Phase2][draft]
 - efficiency of memory usage
- - **additional paragraph in 2.1.2** [Phase2][draft]
+    - **additional paragraph in 2.1.2** [Phase2][draft]
 
 ## Creativity
+
 - collaborative tools (open source, github/google doc)
- - **new chapter on data analysis, visualization, processing, etc.**
+    - **new chapter on data analysis, visualization, processing, etc.**
 - computational artifacts as more than programs
- - **new chapter on data analysis, visualization, processing, etc.**
+    - **new chapter on data analysis, visualization, processing, etc.**
 - computational tools to transform digital media industries such as photoshop, virtual synthesizers, etc
- - **new chapter on data analysis, visualization, processing, etc.**
+    - **new chapter on data analysis, visualization, processing, etc.**
 
 ## Data
+
 - large data sets (databases), queries, filtering
- - **new chapter on data analysis, visualization, processing, etc.**
+    - **new chapter on data analysis, visualization, processing, etc.**
 - data patterns (sequence analysis / cryptography?)
- - **new chapter on data analysis, visualization, processing, etc.**
+    - **new chapter on data analysis, visualization, processing, etc.**
 - collaboration over data (forms, spreadsheets, online multi-edit)
- - **new chapter on data analysis, visualization, processing, etc.**
+    - **new chapter on data analysis, visualization, processing, etc.**
 - data visualization (tables, diagrams)
- - **new chapter on data analysis, visualization, processing, etc.**
+    - **new chapter on data analysis, visualization, processing, etc.**
 - metadata
- - **new chapter on data analysis, visualization, processing, etc.**
+    - **new chapter on data analysis, visualization, processing, etc.**
 
 ## Impact
+
 **Most AP curriculum include specific lesson plans to cover these topic by requiring
```

Comment:
Needs to be either 'topics' or 'this topic'

Reply:
Also, does this need to be in bold? Bold makes things harder to read and this is a relatively long piece of text.